sites/main-site/src/components/event/HackathonProjects/ProjectModal.astro

```diff
@@ -98,28 +98,32 @@ if (!color && category) {
                             )
                         }
                     </div>
-                    {
-                        intro_video && (
-                            <div class="col-lg-4">
-                                <YouTube
-                                    id={intro_video}
-                                    poster={`https://i.ytimg.com/vi/${intro_video}/hqdefault.jpg`}
-                                />
-                            </div>
-                        )
-                    }
-                    {
-                        image && (
-                            <div class="col-lg-4">
-                                <Image
-                                    src={images["../../.." + image]()}
-                                    width="600"
-                                    alt={image_alt ?? ""}
-                                    style="object-fit: contain;"
-                                />
-                            </div>
-                        )
-                    }
+                    {(intro_video || image) && (
+                        <div class="col-lg-4">
+                            {
+                                intro_video && (
+                                    <div class="mb-4">
```

Comment:
```suggestion
                                    <div class="mb-lg-4">
```

Reply:
Should make the layout nicer on mobile

Reply:
Why? I think there should be a margin between the video and the image in any case and for all screen sizes. Since it scales with the screen size, I see no reason to have the margin only for large displays. But we could have a smaller margin, if you prefer that?